app/views/standard/upload.scala.html

```diff
@@ -97,7 +97,7 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
                                 Start upload
                             </button>
 
-                            <a class="govuk-link" href="@routes.HomepageController.homepage()">Cancel</a>
+                            <a class="govuk-link" href="@routes.HomepageController.homepage()">Cancel transfer</a>
```

Comment:
refer to previous comment about being a secondary button rather than a link

Reply:
Have updated these and another I found. Did a visual check on some other of the prototype pages and saw some discrepancies but as this task was explicitly a text update and some of the changes would have an impact on judgement pages, would suggest getting this through and then set up a new task to capture any other css changes.